week-5/A-dom-manipulation/exercise.js

```diff
@@ -23,11 +29,34 @@ Task 3
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
 
+var addParagText = document.querySelector("#addTextBtn");
+var selectButtonsDiv = document.querySelector(".buttons");
+addParagText.addEventListener("click", function() {
+  addSomeText();
+});
 
+function addSomeText() {
+  console.log("heyy");
+  var newParag = document.createElement("p");
+  selectButtonsDiv.appendChild(newParag);
+  newParag.innerText = "Read More!";
```

Comment:
Normally we would want to add the `innerText` before we append the element into the DOM

Reply:
OK. I'll fix it.